src/task_processor/metrics.py

```diff
@@ -0,0 +1,22 @@
+import prometheus_client
+from django.conf import settings
+
+from common.prometheus import Histogram
+
+task_processor_enqueued_tasks_total = prometheus_client.Counter(
```

Comment:
Why is this metric not under the `if` statement below? Do we even need this `if` statement?

Reply:
Clarified this offline - the reason is that the API knows how many tasks it queued, so it should report those metrics.